{
  "document_title": "CVE-2013-6044",
  "document_type": "SUSE CVE",
  "document_publisher": {
    "type": "Vendor",
    "contact_details": "security@suse.de",
    "issuing_authority": "SUSE Security Team"
  },
  "document_tracking": {
    "identification": {
      "id": "SUSE CVE-2013-6044"
    },
    "status": "Interim",
    "version": "1",
    "revision_history": {
      "revision": {
        "number": "5",
        "date": "2022-09-19T00:26:51Z",
        "description": "current"
      }
    },
    "initial_release_date": "2021-05-30T13:15:48Z",
    "current_release_date": "2022-09-19T00:26:51Z",
    "generator": {
      "engine": "cve-database/bin/generate-cvrf-cve.pl",
      "date": "2020-12-27T01:00:00Z"
    }
  },
  "document_notes": [
    {
      "text": "CVE-2013-6044",
      "title": "CVE",
      "type": "Summary"
    },
    {
      "text": "The is_safe_url function in utils/http.py in Django 1.4.x before 1.4.6, 1.5.x before 1.5.2, and 1.6 before beta 2 treats a URL's scheme as safe even if it is not HTTP or HTTPS, which might introduce cross-site scripting (XSS) or other vulnerabilities into Django applications that use this function, as demonstrated by \"the login view in django.contrib.auth.views\" and the javascript: scheme.",
      "title": "Mitre CVE Description",
      "type": "Description"
    },
    {
      "text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
      "title": "Terms of Use",
      "type": "Legal Disclaimer"
    }
  ],
  "document_references": [
    {
      "type": "Self",
      "url": "https://www.suse.com/support/security/rating/",
      "description": "SUSE Security Ratings"
    }
  ],
  "product_tree": {
    "branch": [
      {
        "type": "Product Family",
        "name": "SUSE Cloud 2.0",
        "branch": [
          {
            "type": "Product Name",
            "name": "SUSE Cloud 2.0",
            "full_product_name": {
              "text": "SUSE Cloud 2.0",
              "product_id": "SUSE Cloud 2.0"
            }
          }
        ],
        "full_product_name": {}
      },
      {
        "type": "Product Family",
        "name": "SUSE Cloud 4",
        "branch": [
          {
            "type": "Product Name",
            "name": "SUSE Cloud 4",
            "full_product_name": {
              "text": "SUSE Cloud 4",
              "product_id": "SUSE Cloud 4",
              "cpe": "cpe:/a:suse:suse-cloud:4"
            }
          }
        ],
        "full_product_name": {}
      },
      {
        "type": "Product Family",
        "name": "SUSE Cloud 4 Dependencies",
        "branch": [
          {
            "type": "Product Name",
            "name": "SUSE Cloud 4 Dependencies",
            "full_product_name": {
              "text": "SUSE Cloud 4 Dependencies",
              "product_id": "SUSE Cloud 4 Dependencies",
              "cpe": "cpe:/o:suse:suse-cloud-deps:11:sp3"
            }
          }
        ],
        "full_product_name": {}
      },
      {
        "type": "Product Version",
        "name": "python-django",
        "full_product_name": {
          "text": "python-django",
          "product_id": "python-django"
        }
      }
    ],
    "relationship": [
      {
        "product_reference": "python-django",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "SUSE Cloud 2.0"
      },
      {
        "product_reference": "python-django",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "SUSE Cloud 4"
      },
      {
        "product_reference": "python-django",
        "relation_type": "Default Component Of",
        "relates_to_product_reference": "SUSE Cloud 4 Dependencies"
      }
    ]
  },
  "vulnerability": [
    {
      "notes": [
        {
          "text": "The is_safe_url function in utils/http.py in Django 1.4.x before 1.4.6, 1.5.x before 1.5.2, and 1.6 before beta 2 treats a URL's scheme as safe even if it is not HTTP or HTTPS, which might introduce cross-site scripting (XSS) or other vulnerabilities into Django applications that use this function, as demonstrated by \"the login view in django.contrib.auth.views\" and the javascript: scheme.",
          "title": "Vulnerability Description",
          "type": "General"
        }
      ],
      "cve": "CVE-2013-6044",
      "product_statuses": [
        {
          "type": "Known Not Affected",
          "product_id": [
            "SUSE Cloud 2.0:python-django",
            "SUSE Cloud 4 Dependencies:python-django",
            "SUSE Cloud 4:python-django"
          ]
        }
      ],
      "threats": [
        {
          "type": "Impact",
          "description": "moderate"
        }
      ],
      "cvss_score_sets": {
        "score_set_v_2": {
          "base_score_v_2": "4.3",
          "vector_v_2": "AV:N/AC:M/Au:N/C:N/I:P/A:N"
        },
        "score_set_v_3": {}
      }
    }
  ]
}
